src/web/routes/application/select-address/select-address.js

```diff
@@ -18,6 +19,7 @@ const buildAddressOption = result => ({
 
 const behaviourForGet = () => async (req, res, next) => {
   res.locals.addresses = req.session.postcodeLookupResults.map(buildAddressOption)
+  req.session.nextAllowedStep = '/manual-address'
```

Comment:
We should probably put a comment here explaining why this is necessary.

This also makes me think the paths should be constants somewhere, although not for this PR.

Reply:
Done.